src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -454,7 +479,18 @@ public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verif
             }
             else if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX)
             {
-                return false;
+                // Conditionally enable back package sign verification temporary disabled due to Mozilla drop Symantec as CA on Linux/MAC.
+                // Please note: Linux/MAC case sensitive for env var.
+                string signVerifyEnvVariable = _environmentVariableReader?.GetEnvironmentVariable("DOTNET_OPT_IN_SECURE_PACKAGE_VERIFICATION");
```

Comment:
Given that the constructors that accept `IEnvironmentVariableReader` are all internal, how are customers using the NuGet SDK to call ours APIs directly supposed to opt-in to verification?

For example, [the Arcade SDK has a signature checker](https://github.com/dotnet/arcade/blob/main/src/SignCheck/Microsoft.SignCheck/Verification/NupkgVerifier.cs), which I believe checks that all Microsoft packages are signed by a Microsoft certificate.  If it runs on Linux or Mac, I'm sure they'll still want to verify signatures and not get a false "yes, signature is valid" response because we skipped it.

Reply:
@zivkan 
I fixed it, I saw `dotnet integration test` then realized it.